.circleci/config.yml

```diff
@@ -180,24 +184,21 @@ jobs:
       - restore-cargo-cache
       # Check this to make sure we do not include patched dependencies in
       # the Cargo.lock.
-      - run:
-          command: |
-            [[ -z $(cat Cargo.lock | grep "\[\[patch.unused\]\]") ]]
+      - run: '[ -z $(grep "\[\[patch.unused\]\]" Cargo.lock) ]'
       - run: cargo fmt --all --check
       # TODO: this is incompatible with workspace inheritance, uncomment when
       # https://github.com/DevinR528/cargo-sort/pull/29 is merged
-      # - run: cargo install cargo-sort
+      # - run: # install binstall with script
+      # - run: cargo binstall cargo-sort
       # - run: cargo sort --check --workspace
       - run: |
           cargo clippy --tests \
                        --all-targets \
                        --all-features \
                        --no-deps -- \
-                       --D warnings \
-                       -A clippy::let-unit-value \
-                       -A clippy::format-push-string
```

Comment:
Why have we removed these?

Reply:
For parity with the expectations of the dev who runs plain `cargo clippy`. If we want custom rules, it should be done with config files or a task runner like cargo xtask like we mentioned at some point.